### ISSUE TITLE
Sounds on the right seem centered

### ISSUE THREAD

**Issue by VrydayVrything** [mbillingr/ambisonic#14]
ambisonic = "0.4.1"

```
use ambisonic::{AmbisonicBuilder, HrtfConfig, PlaybackConfiguration};

pub fn test() {
    let mut position = [50.0, -2.0, 0.0];
    let velocity = [-0.1, 0.0, 0.0];

    let config = PlaybackConfiguration::Hrtf(HrtfConfig::default());
    let scene = AmbisonicBuilder::default().with_config(config).build();
    let source = ambisonic::rodio::source::SineWave::new(440);
    let mut sound = scene.play_at(source, position);

    sound.set_velocity(velocity);
    for i in 0..1000 {
        position[0] += velocity[0];
        position[1] += velocity[1];
        position[2] += velocity[2];
        sound.adjust_position(position);
        std::thread::sleep(std::time::Duration::from_millis(10));
    }
}
```

I even put the headphones on backwards to make sure I wasn't just imagining it. Did a blind test with another person as well. Sounds positioned on the right sound centered.

**Comment by mbillingr** [comment]
Thanks for reporting this!

I think your observation is correct, but it's difficult to tell because HRTFs are so subjective.
There may be an issue with the HRTF data, which I scraped from some open dataset. I'll need to investigate that in detail...

Does it sound right for you if you use `PlaybackConfiguration::Stereo` instead?

**Comment by VrydayVrything** [comment]
I did try stereo as well, same observation. (important note in hindsight)

However, I just tested it using audio out on a monitor connected with display port. It works fine... will try my onboard card next.

I usually use a Sound BlasterX G6, which I had the issue on. For the record, I don't have this issue anywhere else... which is odd. (tested on other libraries and I have not noticed it in games, etc.)
